e2e/bzlmod/MODULE.bazel

```diff
@@ -1,18 +1,12 @@
-module(
-    name = "e2e_bzlmod",
-    version = "0.0.0",
-    compatibility_level = 1,
-)
-
-bazel_dep(name = "aspect_rules_js", version = "0.0.0")
-bazel_dep(name = "aspect_bazel_lib", version = "2.7.1")
-bazel_dep(name = "bazel_skylib", version = "1.5.0")
-
+bazel_dep(name = "aspect_rules_js", version = "0.0.0", dev_dependency = True)
```

Comment:
This is because `e2e/bzlmod` doesn't publish rules so this is a dev dependency? Which I guess is true for 99% of projects so e2e tests should do the same?

Reply:
Right. The pattern for a terminal module is that everything is a dev dep because the module is never published and used by other modules. In practice, clients and customers don't mark all their deps as `dev_dependency = True` in the MODULE.bazel files but technically they are. In client & customer repos there is no ambiguity because everything is a dev dep. In our rule sets I think its helpful and clearer to be explicit and it makes it easier to see at glance (and in a search of "bazel_dep") which deps are load bearing to users and which are internal only.